src/rdfTypes/GeometryInfoHelpersImpl.h

```diff
@@ -120,6 +122,12 @@ inline std::string boundingBoxAsWkt(const GeoPoint& lowerLeft,
   return getWKT(box);
 }
 
+// ____________________________________________________________________________
```

Comment:
Add a short comment for each function (although here it is rather self explanatory).